tokio/src/util/trace.rs

```diff
@@ -1,4 +1,31 @@
 cfg_trace! {
```

Comment:
Even though this looks like a macro for internal usage, maybe a little documentation would do good here. 🙂

Reply:
Assuming you’re referring to `poll_op` and `poll_op_inner`, I can’t add documentation because I don’t know what it does in the first place — I just copied it over from the original code (renaming `trace_op` → `poll_op_inner` since that seems like a more appropriate name) 😂

Reply:
IMO `poll_op_inner` is better, indeed, a suggestion may be `trace::poll_operation_with_span!` (I usually rather verbose interfaces, as they help on assuming things, and `op` seems to leave room for ambiguity, so I would replace `op` with `operation`, not sure if this breaks backwards comparability tho) 


Regarding the documentation, see if this makes sense:
```suggestion
//! Tracing macros that help on keeping spans throughout Poll operations

cfg_trace! {
```

Either way, this doc seems relevant regarding `poll_op_inner` https://docs.rs/tracing/latest/tracing/struct.Metadata.html#method.target

Reply:
To be honest, I just don’t really think this is the PR for this, it’s not really relevant to the actual change happening. I agree with most the points though, it’s just probably left for later (and possibly by someone who’s actually used tracing before).

Reply:
Yeah, I agree. This can be done in another PR if necessary.